### PR TITLE
Round 2 of benchmarks

### PR DESCRIPTION
**Pull request by mroth**
TODOs
 - [x] merge #3 #6 #7 with third party contributions (thanks!)
 - [x] update all other repo deps to latest versions
 - [x] update README to acknowledge new platforms
 - [x] migrate old results to legacy results file
 - [x] investigate bootstrap/benchmark scripts
 - [x] standardize run commands via Procfile, use standard port
 - [x] rerun benchmarks on new hardware, update results

### CODE REVIEW

**Comment by hunterboerner** [comment]
Maybe this repo should be renamed to something more generic.

**Comment by mroth** [comment]
Okay!  Results are in. @hunterboerner @hypebeast @chrismccord @josevalim if you want to take a look at the `README` and `RESULTS_v2` files in this branch and give it a look over for any idiocy on my part or places I can provide better explanation, it would be appreciated.

Early next week I'll make any changes to text, import the RESULTS_v2 chart into the README and merge this into master.

**Comment by josevalim** [comment]
@mroth awesome! great work! :heart:

I think it is worth saying that Phoenix is quite close to Rails when it comes in functionality handled by default during such request (the request fetches parameters, session, flash, csrf protection, assets and more) while all the others go with a light-weight approach (and Plug.Router is the lightweight one now!)

**Comment by mroth** [comment]
@josevalim I added a para about that in the new README, was it not clearly written enough, or was it just hard to find amongst all the noise in diff here?  (I'll line comment it inline in case you missed it.)

File: README.md [hunk 1]
```diff
@@ -82,163 +82,41 @@ For that reason, I didn't do too much here. That said, there is probably a lot
 that can/should be disabled in Rails that is there by default for "free" if you
 wanted this to be a closer comparison.  Pull requests welcome if you care!
 
+## Updates!  Round two of benchmarks (January 2015)
```
Comment (mroth) [inline README.md]: @josevalim this is the section I was attempting to address the added functionality of Phoenix in.

Comment (josevalim) [inline README.md]: Oh, I have not read this before, it looks great!

**Pull request merged**